include/glow/Importer/CommonOperatorLoader.h

```diff
@@ -276,14 +276,12 @@ class CommonOperatorLoader : public ProtobufLoader {
     ASSIGN_VALUE_OR_RETURN_ERR(in,
                                getNodeValueOrCreateConstantByName(op.input(0)));
 
-    // This is statically known data, and so we create a Tensor for it and
-    // register it in tensors_.
-    auto *T = new Tensor(ElemKind::Int64ITy, {in.dims().size()});
-    tensors_[opName].reset(T);
-    T->template getHandle<int64_t>() =
+    // This is statically known data, and so we create a Tensor for it.
+    Tensor T(ElemKind::Int64ITy, {in.dims().size()});
+    T.getHandle<int64_t>() =
         std::vector<int64_t>(in.dims().begin(), in.dims().end());
 
-    if (auto resultOrErr = createAndRegisterConstant(opName, *T)) {
+    if (auto resultOrErr = createAndRegisterConstant(opName, std::move(T))) {
```

Comment:
std::move(*T) ???

Reply:
looks legit since now we use `Tensor T` instead of a pointer.